agent/api/status.go

```diff
@@ -31,6 +31,19 @@ func (ts *TaskStatus) String() string {
 	return "UNKNOWN"
 }
 
+// Mapping task status in the agent to that in the backend
+func BackEndStatus(ts *TaskStatus) string {
```

Comment:
I think it makes more sense to have this be a method on TaskStatus. Also, nit, Backend, not BackEnd.